### ISSUE TITLE
Hard to actually choose the grey item

### ISSUE THREAD

**Issue by jidanni** [openstreetmap/iD#7744]
Here (with Foot Path) the user has great difficulty in actually choosing the grey "yes"
![6918-0](https://user-images.githubusercontent.com/1230959/85879884-af19a300-b80d-11ea-851e-a4ab07878479.jpg)

Perhaps the idea is "yes is assumed, so we don't actually offer it, as there is no need to save the tag at all in this case." But this is not clear to the user.

**Comment by maro-21** [comment]
No, it's because _adding access=yes to highway=footway could be read as changing default restrictions (which usually are foot=yes and vehicle=no for highway=footway) to yes, highway=footway + access=yes means "road, which is open for all pedestrians and vehicles"._  https://wiki.openstreetmap.org/wiki/Key:access#Transport_mode_restrictions

Adding `access=yes` to a road is a bad idea, because it marks a feature as wide open to all transport modes Including cars, horses, moped, hazmat and others listed [here](https://wiki.openstreetmap.org/wiki/Key:access#Land-based_transportation).

Access=yes means "all vehicles can use the road, no matter what value `highway` has".

It's clear for the user, because 'yes' is not listed, but anyway many new users type "yes" manually and fill in the other fields.

The problem here is that is assumed to be "no" for footpaths, so the placeholder should be changed.

**Comment by jidanni** [comment]
Well I'm only concerned that among
- item in grey vs. item in black
- item present in blue in drop down list vs. item absent

the user does not grasp that there is deeper logic going on in the several combinations, so maybe the item (e.g., "yes" here) should also appear in the drop down list, but in e.g., red, not blue, and then a mouseover (which hopefully can be machine generated: "A blocks B") appear saying why.

Else the user will think that there must be some bug hindering his mouse from finding all the choices etc.

**Comment by matkoniecz** [comment]
`yes` value is deliberately missing in `access` field
